src/modules/blocks/event-datetime/style.pcss

```diff
@@ -144,3 +144,9 @@
 	display: flex;
 	align-items: center;
 }
+
+/* We need to enforce this rule to override the value set by the SelectControl component. [TEC-4029] */
+
+.tribe-editor__date-time__time-zone-setting {
+    height: auto !important;
```

Comment:
I hate using important...
Couple things - 

- I can reproduce locally (which gives me an edge you didn't have).
- While I suspected it may come from our webpack code, it appears the issue is the class from either the gutenberg plugin - or the core block editor: https://d.pr/i/8cqH4E
- You can get rid of the `!important` by using a more precise selector: https://d.pr/v/849efI
```css
.components-select-control[class*="tribe-editor"] {
    height: auto;
}
```
This has the added benefit of targeting _all_ our selects, in case this is happening somewhere else and we haven't caught it?

CC: @paulmskim

Reply:
Thanks @Camwyn, I've tested the issue using your selector and it works well. I've made the update.